ogorek.go

```diff
@@ -531,10 +535,7 @@ func (d *Decoder) loadUnicode() error {
 		if err != nil {
 			return err
 		}
-		_, err = buf.WriteRune(r)
-		if err != nil {
-			return err
-		}
+		buf.WriteRune(r)
```

Comment:
This change was prepared as separate patch to exactly answer this questions:

https://github.com/kisielk/og-rek/pull/27/commits/d266ce2fe588b146b04dc4316403fbb599abc0e6

To duplicate it here: bytes.Buffer promise not to return error from Write*() ever:

https://golang.org/pkg/bytes/#Buffer.Write
https://golang.org/pkg/bytes/#Buffer.WriteRune
...

("... returning its length and an error, which is always nil but is included to match bufio.Writer's WriteRune. The buffer is grown as needed; if it becomes too large, WriteRune will panic with ErrTooLarge.")

It was already relied upon by ogórek code in other place. Please see details in the patch.

Thanks.